config/initializers/menus.rb

```diff
@@ -42,7 +42,7 @@
             last: true
   menu.push :help, OpenProject::Info.help_url,
             last: true,
-            caption: I18n.t('label_help'),
+            caption: '',
```

Comment:
I might be missing something and also did not try it out, but i believe you just killed accessibility:

It seems that there is no text left that a screen reader can use to explain where the help button leads.